src/calculateRentalCost.js

```diff
@@ -4,7 +4,29 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const SHORT_TERM = 3;
+  const LONG_TERM = 7;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM_DISCOUNT = 50;
+  const PRICE = 40;
+
+  let totalSpent = 0;
+
+  if (days >= LONG_TERM) {
+    totalSpent = days * PRICE - LONG_TERM_DISCOUNT;
+
+    return totalSpent;
+  }
+
+  if (days >= SHORT_TERM) {
+    totalSpent = days * PRICE - SHORT_TERM_DISCOUNT;
+
+    return totalSpent;
```

Comment:
u can return it immediately in both cases(Short and Long conditions)
```suggestion
    return DEFAULT_PRICE - SHORT_TERM_DISCOUNT;
```